src/index.ts

```diff
@@ -63,7 +64,21 @@ export function validationMetadataArrayToSchemas(
             isExcluded(propMeta, options) ||
             isExcluded({ ...propMeta, target }, options)
           )
-      )
+      ).map((propMeta) => {
+            /**
+             * Retrieves all properties that have the Expose decorator from class-transformer
+             * and remaps the property names to the names exposed by the Expose decorator.
+             */
+          const ctMetadata = userOptions?.classTransformerMetadataStorage?.getExposedMetadatas(propMeta.target as any)
```

Comment:
```suggestion
          const exposeMetadata = userOptions?.classTransformerMetadataStorage?.getExposedMetadatas(propMeta.target as any)
```
How's this for more descriptive naming? The constant includes just `Expose` metadata, not all class-transformer metadata.